pkg/buildnum/.tmp_gomock_reflect_199170717/prog.go

```diff
@@ -0,0 +1,46 @@
+
+package main
```

Comment:
I think you added accidentally a temporary file from code generation

Reply:
Actually a whole bunch of them

Reply:
whoops

Reply:
removed them